packages/@aws-cdk/aws-servicecatalog/lib/private/association-manager.ts

```diff
@@ -48,6 +49,19 @@ export class AssociationManager {
     }
   }
 
+  public static associateTagOptions(portfolio: IPortfolio, resourceId: string, tagOptions: TagOptions) {
```

Comment:
1. I don't think `resourceId` as a separate parameter makes too much sense. I think you should remove it, and use `portfolio.portfolioId` in the expression below in `CfnTagOptionAssociation`.
2. Specify the return type of this function explicitly please.